bin/next-dev

```diff
@@ -61,6 +61,16 @@ srv.start(argv.port)
   }
 })
 .catch((err) => {
-  console.error(err)
-  process.exit(1)
+  if (err.code === 'EADDRINUSE') {
+    let errorMessage = `Port ${argv.port} is already in use.`
+    try {
+      const appPackage = JSON.parse(readFileSync(join(dir, 'package.json'), 'utf8'))
```

Comment:
Use [pkg-up](https://www.npmjs.com/package/pkg-up) to find the package.json. Dir could be a directory inside root. In that case, there will be no `package.json` in the `dir`.